src/components/maze/GameboardContainer.tsx

```diff
@@ -412,23 +415,28 @@ export function GameboardContainer({
       </div>
       <div className={styles.mazeContainer} ref={gameboardRef} tabIndex={0}>
         <div className={styles.toolbar}>
-          {earnedButNotMintedCheddar > 0 && (
+          {earnedButNotMintedCheddar > 100 && (
```

Comment:
I forgot to check this. Wait till i do plz.

Reply:
I'm having issues starting a match using local env and bundle-mint branch running. Any idea whay could cause this?
![Uploading Captura de pantalla 2024-12-10 164732.png…]()

Reply:
![Captura de pantalla 2024-12-10 164745](https://github.com/user-attachments/assets/6bc7eaa2-9632-4c67-b64b-4cca8b0b4113)
![Uploading Captura de pantalla 2024-12-10 164732.png…]()

Reply:
![Captura de pantalla 2024-12-10 164732](https://github.com/user-attachments/assets/2afeae25-64e5-42b7-ab33-d7fe557e2f9f)

Reply:
@Mart-Dan-Rossi did you remove the existing data from `/db`, because the structure of data changed after we added base, so maybe that might be the reason?

Reply:
![image](https://github.com/user-attachments/assets/02824906-9dc3-48eb-a3b0-46a18741d2ca)

Reply:
![image](https://github.com/user-attachments/assets/9f663070-043d-4d9c-a3cf-ea8e8ff608dc)

Reply:
Maybe that button should be shown all the time but disabled/with other color until you reach +100

Reply:
The functionallity is ok

Reply:
Also, after clicking the button the amount of cheddar i own is not been changed. Have you check that?

Reply:
yeah good point since I didn't had the keys, I wasn't able to mint, I will call the refresh function after the cheddar is minted (but I hope after refreshing the website, you did see the updated balance) and for the button colour, I think we are having a design of the website altogether, but I will ask Blaze about it and make the changes

Reply:
https://drive.google.com/file/d/1Ps9CdDtGCxzKrZTVn42GeyM0a-OF9Rqo/view?usp=sharing

Reply:
> https://drive.google.com/file/d/1Ps9CdDtGCxzKrZTVn42GeyM0a-OF9Rqo/view?usp=sharing

Yeah, that's what i saw. I was thinking that if we can't make the button work properly BECAUSE we don't have the mint keys this functionallity is flat wong. In production users won't have mint keys.

Reply:
I guess, he can deploy it on testnet (the backend, which has the keys) and then we can test it.